belvedere/src/main/java/zendesk/belvedere/ImageStreamMvp.java

```diff
@@ -8,6 +8,13 @@
 
     interface Model {
 
+        /**
+         * Gets a list of the latest images. This API targets Android Q because of a difference
```

Comment:
I'm not super clear on what this doc is saying. What does `This API targets Android Q` mean to me as an integrator, and if `It will work on all API levels.` do I need to care?

Reply:
most likely not, the comment was from when there was an annotation. I'll remove it

Reply:
done, thanks.